curbs/curbs.yaml

```diff
@@ -0,0 +1,1134 @@
+openapi: 3.1.0
+info:
+  title: Curbs
+  version: "1.0"
+  summary: "The Curbs API is a standard way for cities to digitally publish curb locations and regulations, which can be shared with the public and with companies using curb space. It defines curb policies, curb zones, spaces in zones, and areas around curbs, and is used by Events and Metrics."
```

Comment:
Is a summary and a description valid. Got an error using the Swagger online viewer saying "info: should NOT have additional properties. additionalProperty: summary"

Reply:
I just checked on this and `summary` was introduced in v3.1.0 of the spec, and Swagger only supports up to v3.0.3.

It is probably worth removing `summary` and downgrading the `version` to `3.0.3` for Swagger compatibility. If you agree, there are some other differences between 3.1.0 and 3.0.3 that I can double check on (e.g., `exclusiveMinimum` being a number vs a boolean).

Reply:
I think that's fine, so we can leave as is!